framework/include/actions/Action.h

```diff
@@ -161,6 +161,11 @@ class Action : public ConsoleStreamInterface,
 
   inline bool isParamValid(const std::string & name) const { return _pars.isParamValid(name); }
 
+  inline bool isParamSetByUser(const std::string & name) const
+  {
+    return _pars.isParamSetByUser(name);
+  }
```

Comment:
Inline in the header but outside of the class definition would be my preference but as there are many other methods like this currently, it's not a requirement